zzz_generated/chip-tool/zap-generated/test/Commands.h

```diff
@@ -61334,7 +61334,7 @@ class TestGroupKeyManagementCluster : public TestCommand
 
 private:
     std::atomic_uint16_t mTestIndex;
-    const uint16_t mTestCount = 3;
+    const uint16_t mTestCount = 5;
```

Comment:
This looks wrong.  Was this a hand-edit or something?  This is why the ZAP job is failing.... Please run:
```
./scripts/tools/zap/generate.py -t examples/chip-tool/templates/templates.json -o zzz_generated/chip-tool/zap-generated src/controller/data_model/controller-clusters.zap
```
to fix.

Reply:
done